### ISSUE TITLE
Give menus dropshadows

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6657]
**The problem you're addressing (if any)**
When I open a menu in an app, it typically has an outline—but no dropshadow. This presents a usability problem, with making it difficult to disambiguate what is "in front" vs what is "behind." For reference, I'm on Qubes 4.0.

**Describe the solution you'd like**
Just a dropshadow. 
The mockup below has a `.7 alpha #000000` 10px dropshadow, w/o any x/y offsets. 

<img width="702" alt="Screen Shot 2021-05-31 at 7 47 31 PM" src="https://user-images.githubusercontent.com/8262612/120261525-e8903500-c24c-11eb-9857-550a87d5b0a6.png">

**Where is the value to a user, and who might that user be?**
Sighted users desiring clearer visual hierarchies and focus areas in their UI.

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**
Maybe #6414, but nothing else I was able to see.

**Comment by GWeck** [comment]
This is just what MIcrosft has wrecked in theier design since Windows 8. Lack of a clear separation between fore- and backgroung makes these systems nearly unusable. Qubes is allready far better, and getting still beter with your efforts!

**Comment by p1xxxel** [comment]
@ninavizz Do you mean something like this to be turned on by default?
![Screenshot_2021-12-30_15-47-58](https://user-images.githubusercontent.com/64483930/147742986-905b149b-b0ca-461e-ac79-9dd37900b1da.png)

**Comment by GWeck** [comment]
That's it - exactly!

**Comment by ninavizz** [comment]
YES!! That, as well as the OS menus—but for folks working in apps in qubes, DEFINITELY THAT would be a hyooge "general clarity" usability help to keep brains from cramping! TY! <3

**Comment by p1xxxel** [comment]
actually this was very easy to do. I have done a pull request to enable it by default but this was an option in xfce.
To enable it in your version, click on the top left corner then go to `System Tools` --> `Window Manager Tweaks` --> `Compositor` and then make sure `Show shadows under popup windows` is checked.

If you want to do it through command line, open a dom0 terminal and then paste this :
```bash
xfconf-query -c xfwm4 -p /general/show_popup_shadow -s true
```

This uses [xfconf-query](https://docs.xfce.org/xfce/xfconf/xfconf-query) tool to change the `show_popup_shadow` to true.

**Comment by ninavizz** [comment]
@DemiMarie Was there a _reason_ this hadn't been done before? I seem to remember there being one and am curious, now that this PR has been submitted. 

Thank you @p1xxxel!! I will go do that to my own QubesOS machine, now. :D

**Comment by DemiMarie** [comment]
@ninavizz I am honestly not certain.  Personally I was not aware that this option existed.
